source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -0,0 +1,295 @@
+---
+
+# Process your user’s identity information
+When you [retrieve user information with `/userinfo`](https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/integrate-with-code-flow/#retrieve-user-information), if you [requested identity assurance](https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/choose-the-level-of-identity-confidence/) and the appropriate claims, you’ll receive a response containing additional claims (user attributes). You may receive different claims, depending on how your user proved their identity.
+
+Your service’s needs will determine how you process the other claims that GOV.UK Sign In provides about your user. You’ll probably need to match against information held by your service or organisations you work with.
+
+Most claims are represented by JSON objects. The [core identity claim](#understand-your-user-s-core-identity-claim) is a JSON web token (JWT) protected by an electronic signature for additional security.
+
+You’ll receive a response from `userinfo` that will look similar to this example:
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "sub": "urn:fdc:gov.uk:2022:56P4CMsGh_02YOlWpd8PAOI-2sVlB2nsNU7mcLZYhYw=",
+  "email": "test@example.com",
+  "email_verified": true,
+  "phone": "01406946277",
+  "phone_verified": true,
+  "updated_at":1311280970,
+  "https://vocab.account.gov.uk/v1/coreIdentityJWT": <JWT>,
+  "https://vocab.account.gov.uk/v1/address": [
+    {
+      "uprn": "10022812929",
+      "subBuildingName": "FLAT 5",
+      "buildingName": "WEST LEA",
+      "buildingNumber": "16",
+      "dependentStreetName": "KINGS PARK",
+      "streetName": "HIGH STREET",
+      "doubleDependentAddressLocality": "EREWASH",
+      "dependentAddressLocality": "LONG EATON",
+      "addressLocality": "GREAT MISSENDEN",
+      "postalCode": "HP16 0AL",
+      "addressCountry": "GB",
+      "validFrom": "2022-01-01"
+    },
+    {
+      "uprn": "10002345923",
+      "buildingName": "SAWLEY MARINA",
+      "streetName": "INGWORTH ROAD",
+      "dependentAddressLocality": "LONG EATON",
+      "addressLocality": "NOTTINGHAM",
+      "postalCode": "BH12 1JY",
+      "addressCountry": "GB",
+      "validUntil": "2022-01-01"
+    }
+  ],
+  "https://vocab.account.gov.uk/v1/passport": [
+     {
+        "documentNumber": "1223456",
+        "expiryDate": "2022-02-02"
+      }
+    ]
+}
+```
+## Understand your user’s core identity claim
+The `https://vocab.account.gov.uk/v1/coreIdentityJWT` property in the `/userinfo` response is the core identity claim, which is a JWT representing core identity attributes.
+
+The following are core identity attributes:
+
+* your user's name
+* your user’s date of birth
+* the level of identity confidence GOV.UK Sign In has reached
+
+<%= warning_text("If the <code>https://vocab.account.gov.uk/v1/coreIdentityJWT</code> property is not present, then GOV.UK Sign In was not able to prove your user's identity.") %>
```

Comment:
do we want to give next steps here? I.e. do they try again? Or get in touch?

Reply:
@galund Anything to add?

Reply:
This is instructions for the RP, so they'll never get in touch about a particular user. They will hopefully get in touch if we can't prove _anyone's_ identity _[George smiled weakly]_.

It's up to services what they do if we can't prove the user's identity, but they are all pretty much required to have a non-digital route if nothing else. I don't think we need to mention that here, arguably we could have an entire page where we write about possible strategies in certain unhappy/non-success scenarios, but I think we should wait and see if the Adoption team can prove the user need for that.

Reply:
Thanks, that sounds good to me - one to watch for after MVP.

Reply:
nice - thanks both